src/mbgl/tile/geometry_tile.cpp

```diff
@@ -312,12 +325,8 @@ void GeometryTile::onLayout(std::shared_ptr<LayoutResult> result, const uint64_t
 
     observer->onTileChanged(*this);
 
-    if (layoutResult) {
-        for (const auto& data : layoutResult->layerRenderData) {
-            if (data.second.bucket) {
-                data.second.bucket->check(SYM_GUARD_LOC);
-            }
-        }
+    if (!pending) {
```

Comment:
Did you mean to remove the guard checks?  Merge conflict?

Reply:
Think so, it was unintentional regardless and I've put it back